server/app/controllers/applicant/ApplicantProgramReviewController.java

```diff
@@ -425,14 +426,19 @@ private Result renderIneligiblePage(
               .build();
       return ok(northStarApplicantIneligibleView.render(params)).as(Http.MimeTypes.HTML);
     } else {
-      return ok(
-          ineligibleBlockView.render(
-              request,
-              profile,
-              roApplicantProgramService,
-              messagesApi.preferred(request),
-              applicantId,
-              programDefinition));
+      try {
```

Comment:
**question:** Same here - I don't think this needs to be wrapped in a try/ catch?

Reply:
same as above.